### PR TITLE
Add Jaeger, OTel, OpenTelemetry, and Zipkin to dictionary

### PR DESCRIPTION
**Pull request by jdbaldry**

### CODE REVIEW

File: vale/dictionaries/en_US-grafana.dic [hunk 1]
```diff
@@ -1,13 +1,16 @@
-38
+42
 API/S po:noun
 Datadog/ po:noun
 ESLint/ po:noun
 Goldmark/ po:noun
 Grafana/M po:noun
+Jaeger/ po:noun
```
Comment (JStickler) [inline vale/dictionaries/en_US-grafana.dic:7]: FYI - Jaeger can also be an adjective (Jaeger Operator, Jaeger client, Jaeger agent)

File: vale/dictionaries/en_US-grafana.dic [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 JMeter/ po:noun
 JUnit/ po:noun
 OAuth/ po:noun
+OTel/ po:noun
```
Comment (JStickler) [inline vale/dictionaries/en_US-grafana.dic:11]: Same with OTel (OTel API, Otel collector, etc.)

File: vale/dictionaries/en_US-grafana.dic (hunk 1 above)
Comment (jdbaldry) [reply vale/dictionaries/en_US-grafana.dic:7]: Great point! I think I have to add them as separate entries so will do that.

Comment (jdbaldry) [reply vale/dictionaries/en_US-grafana.dic:7]: I think there is a few other places in the word list where this needs to be added also so I've opened: https://github.com/grafana/writers-toolkit/pull/369

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**